file-formats/sito/type/zif_aff_sito_v1.intf.abap

```diff
@@ -0,0 +1,551 @@
+      "! <p class="shorttext">Version</p>
+      "! Service Version for Value Help
+      version                     TYPE n LENGTH 4,
+      "! <p class="shorttext">Protocol Version</p>
+      "! Service Protocol Version for Value Help
+      "! $required
+      protocol_version            TYPE ty_sit2_do_vh_srv_protocol_ver,
+      "! <p class="shorttext">Path Type</p>
+      "! Service Path Type for Value Help
+      "! $required
+      path_type                   TYPE ty_sit2_do_vh_srv_path_type,
+      "! <p class="shorttext">Scope</p>
+      "! Source
+      "! $required
+      scope                       TYPE ty_sit2_do_scope,
+      "! <p class="shorttext">Definition</p>
+      "! Service Definition for Value Help
+      definition                  TYPE c LENGTH 30,
+      "! <p class="shorttext">Binding</p>
+      "! Service Binding for Value Help
+      binding                     TYPE c LENGTH 40,
+      "! <p class="shorttext">ID</p>
+      "! Service ID for Value Help
+      id                          TYPE c LENGTH 32,
+      "! <p class="shorttext">Custom Path</p>
+      "! Service Custom Path for Value Help
+      custom_path                 TYPE c LENGTH 255,
+      value_help_service_provider TYPE ty_sit2_obj_vh_s_p_list,
+    END OF ty_sit2_obj_vh_s,
+
+    "! <p class="shorttext">Value Help Service</p>
+    "! Value Help Service for Situation Object
+    ty_sit2_obj_vh_s_list TYPE STANDARD TABLE OF ty_sit2_obj_vh_s WITH DEFAULT KEY,
+
+    "! <p class="shorttext">Situation Object</p>
+    "! Situation Object
+    BEGIN OF ty_sit2_object,
+      "! <p class="shorttext">Extensible</p>
+      "! Object Extensible
+      extensible                  TYPE abap_bool,
+      "! <p class="shorttext">Reusable</p>
+      "! Object Reusable
+      reusable                    TYPE abap_bool,
+      "! <p class="shorttext">Scope</p>
+      "! Source
+      "! $required
+      scope                       TYPE ty_sit2_do_scope,
+      "! <p class="shorttext">SAP Object Type</p>
+      "! SAP Object Type
+      sap_object_type             TYPE c LENGTH 30,
+      situation_object_text       TYPE ty_sit2_object_t_list,
+      situation_object_structure  TYPE ty_sit2_obj_struc_list,
+      situation_object_event      TYPE ty_sit2_obj_events_list,
+      situation_object_action     TYPE ty_sit2_obj_act_list,
+      situation_object_navigation TYPE ty_sit2_obj_nav_list,
+      situation_object_callback   TYPE ty_sit2_obj_cb_list,
+      value_help_service          TYPE ty_sit2_obj_vh_s_list,
+    END OF ty_sit2_object,
+
+
+
+    "! <p class="shorttext">Situation Object Properties</p>
+    "! Situation Object properties
+    BEGIN OF ty_main,
+      "! <p class="shorttext">Format version</p>
+      "! Format version
+      "! $required
+      format_version   TYPE zif_aff_types_v1=>ty_format_version,
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header           TYPE zif_aff_types_v1=>ty_header_60,
```

Comment:
Are situation objects relevant for Key User Extensibility (`"abapLanguageVersion": "keyUser"`)? If this is not the case you should change the type to `zif_aff_types_v1=>ty_format_version_cloud`.

```suggestion
      header           TYPE zif_aff_types_v1=>ty_header_60_cloud,
```

Reply:
Replied here: https://github.com/SAP/abap-file-formats/pull/380#issuecomment-1196620771

Reply:
Most object types don't offer key user extensibility. This was the reason why I asked. If you offer key user extensibility as mentioned in https://github.com/SAP/abap-file-formats/pull/380#issuecomment-1196620771, `ty_header_60` is fine. 👍